src/main/generic/Core.js

```diff
@@ -1,9 +1,9 @@
 class Core {
-    constructor() {
-        return this._init();
+    constructor(options) {
+        return this._init(options);
     }
 
-    async _init() {
+    async _init({ walletSeed }) {
```

Comment:
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

Search for: _Unpacking fields from objects passed as function parameter_